kolibri/plugins/learn/assets/src/views/TopicsPage/index.vue

```diff
@@ -31,7 +35,10 @@
       />
 
       <!-- mobile tabs (different alignment and interactions) -->
-      <TopicsMobileHeader v-else :topic="topic" />
```

Comment:
Not sure what could be wrong here. But I do normally run the linting command which almost every time reverts the IDE formatting. Any reason this could be happening?

Reply:
I think it's more that the way your IDE has linted it is still acceptable (but so is the original) so your IDE is being _more_ opinionated than the Kolibri linting in this case.